tee-worker/omni-executor/executor-crypto/src/aes256.rs

```diff
@@ -21,17 +21,27 @@ pub type Aes256KeyNonce = [u8; NONCE_LEN];
 // metadata that is required for decryption
 //
 // by default a postfix tag is used => last 16 bytes of ciphertext is MAC tag
-#[serde_as]
-#[derive(Debug, Default, Clone, Eq, PartialEq, Encode, Decode, Serialize, Deserialize)]
+#[derive(Debug, Default, Clone, Eq, PartialEq, Encode, Decode)]
 pub struct AesOutput {
-	#[serde_as(as = "serde_with::hex::Hex")]
 	pub ciphertext: Vec<u8>,
-	#[serde_as(as = "serde_with::hex::Hex")]
 	pub aad: Vec<u8>,
-	#[serde_as(as = "serde_with::hex::Hex")]
 	pub nonce: Aes256KeyNonce, // IV
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+
+pub struct SerdeAesOutput {
```

Comment:
We might end up with many `Serde*` types in the future if we take this approach that's why I prefere `serde_with` way which only tackles this cosmetic issue described in PR.